bin/setup

```diff
@@ -19,6 +19,23 @@ echo
 
 bundle install --binstubs
 
+mh_ui_testing_release=1.0.0
```

Comment:
could this be dumped into a requirements.txt or whatever? This doesn't seem idiomatic.